src/bosh-google-cpi/integration/stemcell_test.go

```diff
@@ -29,12 +53,24 @@ var _ = Describe("Stemcell", func() {
 			  "method": "delete_stemcell",
 			  "arguments": ["%v"]
 			}`, stemcellCID)
+		assertSucceeds(request)
+	})
 
-		response, err := execCPI(request)
-		Expect(err).ToNot(HaveOccurred())
-		Expect(response.Error).To(BeNil())
-		Expect(response.Result).To(BeNil())
-
+	//This currently fails due to issue: https://github.com/google/google-api-go-client/issues/181
```

Comment:
Correct, right now the SHA1 is unused as GCP does not support checksum validation yet. If you could find out the status of that feature, I'm sure that would help @cppforlife sleep a little easier :)